src/lib/pickles/pickles.ml

```diff
@@ -2867,15 +3007,9 @@ module Make_str (_ : Wire_types.Concrete) = struct
         ; fork = None
         }
 
-      let tag =
-        let tagname = "" in
-        Tag.create ~kind:Compiled tagname
-
-      let rule : _ Inductive_rule.t =
-        let open Impls.Step in
+      let rule self : _ Inductive_rule.t =
```

Comment:
what was `tag` and why is it not required any more?

Reply:
Previously, this test expanded `Pickles.compile` inline; now, we replace that with the true `Pickles.compile`, overriding only the parts that are relevant to the adversarial test (ie. stubbing out `wrap_main` and modifying the witness).